.github/workflows/docbuild-and-upload.yml

```diff
@@ -37,6 +37,14 @@ jobs:
       run: |
         source activate pandas-dev
         python web/pandas_web.py web/pandas --target-path=web/build
+
+    - name: Check building docs with --single works
```

Comment:
I think this would be better suited in a dedicated place for testing doc building since this workflow appears to be more of the "production" docbuild flow. Maybe this can be tested in `scripts/tests`